### ISSUE TITLE
train on hmdb51

### ISSUE THREAD

**Issue by FelixZhang7** [kenshohara/3D-ResNets-PyTorch#93]
I want to train this resnet on hmdb51 dataset i,i use     *python main.py --root_path ~/3D-ResNets-PyTorch --video_path data/hmdb51_videos/jpg --annotation_path hmdb51_1.json --result_path results --dataset hmdb51 --model resnet --model_depth 34 --n_classes 51 --batch_size 128 --n_threads 16 --checkpoint 20*
but i got  0 acc and 0 loss in train.log and train_batch.log,Can anyone help? Thanks in advance.
